boltdesignsystem/bolt
# DS-565 My Quick Links 2.0: Manage My Quick Links page (UX Mockup)

## Jira

https://pegadigitalit.atlassian.net/browse/DS-565

## Summary

A mockup of "My Quick Links 2.0" was created in the Pattern Lab sandbox.

## Details

#### Desktop:

- two columns are created with a list of links
- ~~visible scrollbar is added~~ (feedback request: add default scrollbar)
- skip links are added to switch between columns
- ~~aria-describedby is added to make items more accessible~~ (Changed to invisible descriptive`span`s)
- spaces between list items are increased
- descriptive titles of the columns are created
- tooltips are removed
- external link icons were added

#### Mobile

- "add Items to my quick links" column is switched to modal
- ~~fixed bottom bar with the modal trigger is added~~ (feedback request: modal trigger is static above the list)
- skip links are triggers to switch between modal and the list

## How to test

Pull the branch. Check the mockup if it's accurate to the sent wireframes. Check if UX and its elements are accessible.
